internal/pkg/config/workload.go

```diff
@@ -142,6 +116,40 @@ func (s *Store) GetJob(appName, jobName string) (*Workload, error) {
 	return &job, nil
 }
 
+// GetWorkload gets a job or service belonging to an application by name.
+func (s *Store) GetWorkload(appName, name string) (*Workload, error) {
+	param, err := s.getWorkloadParam(appName, name)
+	if err != nil {
+		return nil, fmt.Errorf("get job or service: %w", err)
```

Comment:
ugh I am struggling with the fact that we're writing "job or service" everywhere and will find it difficult to scale.
Sorry for flip-flopping on this but let's just change the error to `"get workload:"` since we already write `"workload type"` in the `init` cmd